sos/collector/clusters/ocp.py

```diff
@@ -69,7 +69,8 @@ class ocp(Cluster):
         ('role', 'master', 'Colon delimited list of roles to filter on'),
         ('kubeconfig', '', 'Path to the kubeconfig file'),
         ('token', '', 'Service account token to use for oc authorization'),
-        ('with-api', False, 'Collect OCP API data from a master node')
+        ('with-api', False, 'Collect OCP API data from a master node'),
+        ('api-url', False, 'Collect OCP API data from a master node'),
```

Comment:
If `api-url` is meant to be string, shouldn't it have `''` as default instead of `False`?